cmd/cli/install_test.go

```diff
@@ -351,7 +351,7 @@ var _ = Describe("Running the install command", func() {
 		})
 	})
 
-	Describe("when a mesh with the given name already exists", func() {
+	Describe("when a mesh with the given name already exists and enforceSingleMesh is false", func() {
```

Comment:
It would be good if we can add tests for the scenarios you manually tested as well

Reply:
There were multiple tests testing various cases of enforceSingleMesh, merged them into one test function